packages/gelbooru/src/index.ts

```diff
@@ -44,11 +44,15 @@ namespace GelbooruImageSource {
     endpoint: string
   }
 
-  export const Config: Schema<Config> = Schema.object({
-    label: Schema.string().default('gelbooru').description('图源标签，可用于在指令中手动指定图源。'),
-    weight: Schema.number().min(1).default(1).description('图源权重。在多个符合标签的图源中，将按照各自的权重随机选择。'),
-    endpoint: Schema.string().description('gelbooru 的 URL。').default('https://gelbooru.com/index.php'),
-  })
+  export const Config: Schema<Config> = Schema.intersect([
+    Schema.object({
+      label: Schema.string().default('gelbooru').description('图源标签，可用于在指令中手动指定图源。'),
+      weight: Schema.number().min(1).default(1).description('图源权重。在多个符合标签的图源中，将按照各自的权重随机选择。'),
+    }).description('全局设置'),
+    Schema.object({
+      endpoint: Schema.string().description('gelbooru 的 URL。').default('https://gelbooru.com/index.php'),
```

Comment:
```suggestion
      endpoint: Schema.string().description('Gelbooru 的 URL。').default('https://gelbooru.com/index.php'),
```

Use upper case if refer to the site itself